src/ipc/ipc4/handler.c

```diff
@@ -251,6 +251,36 @@ static struct ipc_comp_dev *pipeline_get_host_dev(struct ipc_comp_dev *ppl_icd)
 	struct ipc *ipc = ipc_get();
 	int host_id;
 
+	/**
```

Comment:
Stylistic note. we haven't really used this kind of C++/java notation of explanations in the function body. I'd stick with existing style and open with "/*<newline>Synchronize the direction...". Not a biggie, but in case you anyways update this, Otherwise future contributors don't know which is the correct syntax and we'll just having different convetions more and more mixed.